### PR TITLE
SONAR-6717 Drop support of plugin extension issue action

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

File: sonar-plugin-api/src/main/java/org/sonar/api/issue/action/Actions.java [hunk 1]
```diff
@@ -25,7 +25,9 @@
 
 /**
  * @since 3.6
+ * @deprecated in 5.5. Webapp cannot be customized anymore to define actions on issues
  */
+@Deprecated
 @ServerSide
 public class Actions {
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/issue/action/Actions.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)

File: sonar-plugin-api/src/main/java/org/sonar/api/issue/action/Action.java [hunk 2]
```diff
@@ -30,7 +30,9 @@
 
 /**
  * @since 3.6
+ * @deprecated in 5.5. Webapp cannot be customized anymore to define actions on issues
  */
+@Deprecated
 public class Action {
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/issue/action/Action.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* INFO 2 info

Watch the comments in this conversation to review them.

**Comment by julienlancelot** [comment]
LGTM

**Pull request closed without merging**